.github/workflows/verify.yml

```diff
@@ -1,5 +1,21 @@
 name: Verify
 
+# https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#permissions
+permissions:
+  actions: none
+  checks: none
```

Comment:
Feel this is part of the checks as well so I question this choice, might deserve further checking.

Reply:
As above - https://github.com/rapid7/metasploit-framework/pull/16162/files#r805056661